### PR TITLE
Extend logs delegation contract

### PR DESCRIPTION
**Pull request by miiu96**
## Reasoning behind the pull request
- Extended logs generated by the `delegation` contract
  
## Proposed changes
- Extended the logs that are generated by `delegation` smart contract in case of `unDelegate` or `withdraw` 
- Upgraded the indexer version
  PR from indexer: https://github.com/multiversx/mx-chain-es-go/pull/196

## Testing procedure
- Create a `delegation` contract.
- Do `delegate` from another address.
- Do multiple `unDelegate` operations in multiple epochs from the same address.
- Check in the `delegators` index from the Elasticsearch database if there is a list of all the `unDelegate` operations for the test address.   

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/ElrondNetwork/elrond-go
 go 1.17
 
 require (
-	github.com/ElrondNetwork/elastic-indexer-go v1.3.6
+	github.com/ElrondNetwork/elastic-indexer-go v1.3.7-0.20230110091622-85f314f36285
```
Comment (iulianpascalau) [inline go.mod:6]: proper release

File: vm/systemSmartContracts/logs.go [hunk 2]
```diff
@@ -30,9 +30,12 @@ func (d *delegation) createAndAddLogEntryForWithdraw(
 	delegator *DelegatorData,
 	numUsers uint64,
 	wasDeleted bool,
+	withdrawFundKeys [][]byte,
 ) {
 	activeFund := d.getFundForLogEntry(delegator.ActiveFund)
-	d.createAndAddLogEntryCustom(function, address, actualUserUnBond.Bytes(), activeFund.Bytes(), big.NewInt(0).SetUint64(numUsers).Bytes(), globalFund.TotalActive.Bytes(), boolToSlice(wasDeleted))
+	topics := append([][]byte{}, actualUserUnBond.Bytes(), activeFund.Bytes(), big.NewInt(0).SetUint64(numUsers).Bytes(), globalFund.TotalActive.Bytes(), boolToSlice(wasDeleted))
```
Comment (iulianpascalau) [inline vm/systemSmartContracts/logs.go:36]: `topics := append(make([][]byte, 0), ........` as to avoid empty slice declaration using a literal?

Comment (miiu96) [reply vm/systemSmartContracts/logs.go:36]: changed

File: go.mod (hunk 1 above)
Comment (miiu96) [reply go.mod:6]: will add

**Review by gabi-vuls — APPROVED** [review]
System test passed

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**